pkg/operation/botanist/component/vpnshoot/vpnshoot.go

```diff
@@ -565,6 +574,163 @@ func (v *vpnShoot) computeResourcesData(secretCAVPN, secretVPNShoot *corev1.Secr
 		clusterRolePSP,
 		roleBindingPSP,
 	)
+	return registry.AddAllAndSerialize(objects...)
+}
+
+func (v *vpnShoot) podDisruptionBudget() (client.Object, error) {
+	var (
+		pdbObjectMeta = metav1.ObjectMeta{
+			Name:      deploymentName,
+			Namespace: metav1.NamespaceSystem,
+			Labels:    getLabels(),
+		}
+		pdbMaxUnavailable = intstr.FromInt(1)
+		pdbSelector       = &metav1.LabelSelector{MatchLabels: getLabels()}
+	)
+
+	seedK8sVersionGreaterEqual121, err := version.CompareVersions(v.shootVersion, ">=", "1.21")
```

Comment:
`seedK8sVersionGreaterEqual121` sounds wrong 😉 But now that the `shootVersion` is `*semver.Version`, you can use https://github.com/gardener/gardener/blob/6a0fea86850ffec8937d1956bdf1a8ca6d074f3b/pkg/utils/version/version.go#L25-L54

Reply:
Thanks, applied and simplified